src/changes/changes.xml

```diff
@@ -31,6 +31,19 @@
          - "remove" - Removed
     -->
     <release version="3.0.0" date="2021-MM-DD" description="GA Release 3.0.0">
+      <action issue="LOG4J2-3024" dev="mattsicker" type="fix" due-to="Tim Perry">
+        Cleanly shutdown status logger when log4j is stopping to ensure file handles
+        are promptly released to avoid resource leakage.
+      </action>
```

Comment:
Would you mind including only your changes, please? (Is Matt, @jvz, really responsible for this change?)

Reply:
The `dev` attribute value should be filled with whoever merges the patch.

Reply:
What is the correct thing for me to put in changes.xml?

<action issue="LOG4J2-3024" dev="UPDATE-ME" type="fix" due-to="Tim Perry">